content/BindingMethods/BindingMethods.q.md

```diff
@@ -0,0 +1,1601 @@
+
+    $pstmt = $conn->prepare($sql);
+    $pstmt->bindParam(':min_birth_date', $data['min_birth_date'], PDO::PARAM_STR);
+    $pstmt->bindParam(':max_birth_date', $data['max_birth_date'], PDO::PARAM_STR);
+    $pstmt->execute();
+    $data = $pstmt->fetchAll();
+
+    foreach ($data as $row) {
+        echo $row['student_id'].$row['first_name'].$row['last_name'].$row['birth_date'].$row['gender']. ' ';
+    }
+?>
+```
+/// type=SS, answer=[5] 
+
+Execute the program. What is its output?
+
+- It produces an error.
+
+- No output is displayed.
+
+- It prints `Unable to establish a connection.`.
+
+- It prints `Successfully connected to the database.` and `7CarloPears1998-04-04M 8GenevieLieser1996-05-25F`.
+
+- It prints `Successfully connected to the database.` and `1JohnSmith1999-02-10M 2SamanthaDanes1999-10-12F 6AlisaElls1999-06-30F`.
+
+
+/// type=MS, answer=[2,3]
+
+Which of the following are array values in the associative array `$data`?
+
+- `students`
+
+- `1999-01-01`
+
+- `2000-01-01`
+
+- `min_birth_date`
+
+- `max_birth_date`
+
+
+/// type=SS, answer=[4]
+
+Which statement best describes the code on lines 4, 5, 6, and 7 of `student.php`?
+
+- It displays the elements of the associative array `$data`.
+
+- It holds two keys with one value in each key such as `1999-01-01` and `2000-01-01`.
+
+- It holds the elements of the associative array `$data` using the `array()` language construct.
+
+- It creates the associative array named `$data` with two keys `min_birth_date` and `max_birth_date`.
+
+- It accesses the elements of `$data` using the `array()` language construct and the short array syntax `[]`.
+
+
+/// type=MS, answer=[4,5]
+
+In `student.php`, which of the following are named placeholders?
+
+- `students`
+
+- `birth_date`
+
+- `min_birth_date`
+
+- `:max_birth_date`
+
+- `:min_birth_date`
+
+
+/// type=SS, answer=[1]
+
+In `student.php`, what value is associated with the named placeholder `:min_birth_date` on line 9?
+
+- `1999-01-01`
+
+- `2000-01-01`
+
+- `birth_date`
+
+- `min_birth_date`
+
+- `max_birth_date`
+
+
+/// type=SS, answer=[2]
+
+In `student.php`, what value is associated with the named placeholder `:max_birth_date` on line 9?
+
+- `1999-01-01`
+
+- `2000-01-01`
+
+- `birth_date`
+
+- `min_birth_date`
+
+- `max_birth_date`
+
+
+/// type=SS, answer=[5]
+
+Which statement best describes `WHERE birth_date >= :min_birth_date AND birth_date < :max_birth_date` on line 9 of `student.php`?
+
+- It selects the rows in the `students` table related to the values `1999-01-01` and `2000-01-01`.
+
+- It selects the rows in the `students` table equivalent to the dates `1999-01-01` and `2000-01-01`.
+
+- It returns all records in the `students` table whose `birth_date` starts with `1999-01-01` or `2000-01-01`.
+
+- It updates the rows in the `students` table whose `birth_date` is between the dates `1999-01-01` and `2000-01-01`.
```

Comment:
I have notice that in our option statements here we use `the values` and `the dates` that refers to similar values. Is it possible to rephrase the statement to have a consistent term to use. How about removing the phrase `the dates` from the statements? Please check. Thanks.